lib/shopify_api/resources/resource_feedback.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ShopifyAPI
+  class ResourceFeedback < Base
+    class ExistingFeedbackSaved < StandardError; end
+    include DisablePrefixCheck
+
+    STATE_SUCCESS = 'success'
+    STATE_REQUIRES_ACTION = 'requires_action'
```

Comment:
we've never used this pattern of string constants before, but i'm not opposed to it